nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/dto/ApprovalStatusDto.java

```diff
@@ -0,0 +1,28 @@
+package no.sikt.nva.nvi.common.service.dto;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import no.sikt.nva.nvi.common.service.model.ApprovalStatus;
+
+public enum ApprovalStatusDto {
+
+    NEW("New"), PENDING("Pending"), APPROVED("Approved"), REJECTED("Rejected");
+
+    @JsonValue
+    private final String value;
+
+    ApprovalStatusDto(String value) {
+        this.value = value;
+    }
+
+    public static ApprovalStatusDto from(ApprovalStatus status, boolean approvalHasAssignee) {
+        return switch (status) {
+            case PENDING -> approvalHasAssignee ? PENDING : NEW;
+            case APPROVED -> APPROVED;
+            case REJECTED -> REJECTED;
+        };
+    }
```

Comment:
What about sending the approval from the method, since both the status and the boolean are parts of the approval:
```
public static ApprovalStatusDto from(Approval approval) {
        return switch (approval.getStatus()) {
            case PENDING -> approval.isAssigned() ? PENDING : NEW;
            case APPROVED -> APPROVED;
            case REJECTED -> REJECTED;
        };
    }
```